### PR TITLE
docs: update IR documentation

### PR DESCRIPTION
**Pull request by Ali-Abikhodr**

### CODE REVIEW

File: src/book/spectra/ir/view-and-process-ir/IRCompare/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+## Compare IR spectra
+
+You can compare multiple IR spectra by first opening up an IR spectrum, and then clicking "Compare" on top of the IR spectrum.
+
+<img src="compare.png">
+
+Afterwhich, under __List of selected enteries__ (1), you can choose the other molecule which has the spectrum that you would like to compare. Then choose the IR spectrum you would like to be plotted under __List of IR spectra__ (2).
```
Comment (kjappelbaum) [inline src/book/spectra/ir/view-and-process-ir/IRCompare/index.md:7]: please fix the typos

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
Looking at the other IR tips/documentation pages it seems that there are some typos and that some of them could still use a n image. Would be great if you could add something there.

File: src/book/spectra/ir/view-and-process-ir/IRCompare/index.md (hunk 1 above)
Comment (Ali-Abikhodr) [reply src/book/spectra/ir/view-and-process-ir/IRCompare/index.md:7]: hello, i did a while back and i thought that commiting on visual studio will automatically update my older pull rquest, sorry it is now 2 pull requests

**Comment by kjappelbaum** [comment]
duplicated in #26

**Pull request closed without merging**